### DISCUSSION TITLE
Gitbash appends prefixes to absolute posix paths in command arguments

### DISCUSSION THREAD

**Discussion by berthin** [git-for-windows/git#3038] [Q&A]
Did someone experience this?

```
$ echo "import sys; print(sys.argv)" > test.py
$ python test.py test /dev/de
['test.py', 'test', 'C:/Program Files/Git/dev/de']
```

Gitbash is adding the `C:/Program Files/Git` prefix when the command argument is a posix path, for my case the workaround is to use double slash `//dev/de`, but I am wondering if that is a misconfiguration or if it could be solved somehow.

PS. I dont know how to get the gitbash version, all I can provide is my git version, dont know if that's the same.
```
$  git --version --build-options
git version 2.20.1.windows.1
cpu: x86_64
built from commit: 7c9fbc07db0e2939b36095df45864b8cda19b64f
sizeof-long: 4
sizeof-size_t: 8
```

**Comment by dscho** [comment]
See https://github.com/git-for-windows/build-extra/blob/master/ReleaseNotes.md#known-issues:

> * If you specify command-line options starting with a slash, POSIX-to-Windows path conversion will kick in converting e.g. "`/usr/bin/bash.exe`" to "`C:\Program Files\Git\usr\bin\bash.exe`". When that is not desired -- e.g. "`--upload-pack=/opt/git/bin/git-upload-pack`" or "`-L/regex/`" -- you need to set the environment variable `MSYS_NO_PATHCONV` temporarily, like so:
>
>  > `MSYS_NO_PATHCONV=1 git blame -L/pathconv/ msys2_path_conv.cc`
>
>  Alternatively, you can double the first slash to avoid POSIX-to-Windows path conversion, e.g. "`//usr/bin/bash.exe`".
